xml/vt_docker_quick.xml

```diff
@@ -328,7 +328,7 @@
    </para>
 <!--taroth 2015-07-02: http://doccomments.provo.novell.com/comments/28724 -->
    <procedure>
-    <title>Enabling the Container Module</title>
+    <title>Enabling the Container Module - interactive way</title>
```

Comment:
I think that technically speaking it YaST should use the same set of libraries of SUSEConnect, it shouldn't be invoking SUSEConnect. I proposed these new titles because they seem more concise to me. But as you know it's always a matter of personal taste